agent/consul/acl_replication.go

```diff
@@ -484,11 +484,12 @@ func (s *Server) IsACLReplicationEnabled() bool {
 		s.config.ACLTokenReplication
 }
 
-func (s *Server) updateACLReplicationStatusError() {
+func (s *Server) updateACLReplicationStatusError(errorMsg error) {
```

Comment:
Since we are already doing a `err != nil` check in the caller, we were thinking it would be safer to accept a string here instead of an error. That way no new callers can accidentally pass a `nil` error.

```suggestion
func (s *Server) updateACLReplicationStatusError(errorMsg string) {
```

which I guess will requires a change to the line below that uses `errorMsg`.

Reply:
Done.